src/Module.php

```diff
@@ -29,9 +29,11 @@ public function getConfig()
      */
     public function onBootstrap(MvcEvent $e)
     {
-        $app            = $e->getTarget();
+        $app = $e->getTarget();
+        /** @psalm-suppress PossiblyNullReference */
         $serviceManager = $app->getServiceManager();
-        $eventManager   = $app->getEventManager();
+        /** @psalm-suppress PossiblyNullReference */
```

Comment:
Shouldn't be suppressed: should expand baseline instead

Reply:
Changed